docs/spec/auth/token.md

```diff
@@ -151,7 +149,7 @@ challenge, the client will need to make a `GET` request to the URL
     </dd>
 </dl>
 
-#### Example Token Request
+#### Example
```

Comment:
Calling this section "Example ..." might not be the best thing to do as the whole section starting from "How to authenticate" above is really the beginning of the full example workflow. This section can probably be sub-titled "Getting a Bearer Token" or something similar.